tests/acceptance/features/apiNotification/deprovisioningNotification.feature

```diff
@@ -169,3 +169,71 @@ Feature: Deprovisioning notification
       | Space Admin |
       | User        |
       | User Light  |
+
+
+  Scenario Outline: administrator creates a deprovisioning notification with different date formats
+    When the administrator creates a deprovisioning notification for date "<deprovision_date>" of format "<deprovision_date_format>"
+    Then the HTTP status code should be "200"
+    When user "Alice" lists all notifications
+    Then the HTTP status code should be "200"
```

Comment:
instead of double when then use And and check both status with single step.Same apply for the down scenario.